openlibrary/core/vendors.py

```diff
@@ -187,6 +206,16 @@ def clean_amazon_metadata_for_load(metadata):
     if metadata.get('source_records'):
         asin = metadata.get('source_records')[0].replace('amazon:', '')
         conforming_metadata['identifiers'] = {'amazon': [asin]}
+    title, subtitle = split_amazon_title(metadata['title'])
+    conforming_metadata['title'] == title
+    if subtitle:
+        conforming_metadata['full_title'] = title + ' : ' + subtitle
```

Comment:
I think this should be set to `title` outside of this `if` statement; that way `full_title` is always defined.

Reply:
I was avoiding setting `full_title` unless it was different from `title`, which seems to be the current way of doing things. I might leave this until we have a clearer picture of how `full_title` is used.